src/software/networking/proto_multicast_sender.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <boost/asio.hpp>
+#include <boost/bind.hpp>
+#include <string>
+
+template <class SendProto>
+class ProtoMulticastSender
+{
+   public:
+    /**
+     * Creates a ProtoMulticastSender that sends the SendProto over the network on the
+     * given address and port.
+     *
+     * @param io_service The io_service to use to service outgoing SendProto data
+     * @param ip_address The ip address of the multicast group to send data on
```

Comment:
If you make `io_service` refer to `io_service`, it's inconsistent to make `ip address` refer to `ip_address` in these two lines.

Reply:
I went to go change it but I realized in a way it makes sense because an `io_service` is a boost concept, an abstraction around the OS to perform networking operation. So we can't really call it an `io service`. 

And the `ip_address` is the name of the argument used to send an ip address. So changing that to `The ip_address` isn't entirely correct either.